Book-A-Desk.Api/EmailNotification.fs

```diff
@@ -0,0 +1,46 @@
+namespace Book_A_Desk.Api
+
+open System
+open System.Net.Mail
+open Book_A_Desk.Api.Models
+open Book_A_Desk.Domain.QueriesHandler
+open Book_A_Desk.Domain.Office.Domain
+
+type EmailNotification =
+    {
+        SendEmailNotification: Booking -> unit
+    }
+
+module EmailNotification =
+    let initialize getEmailServiceConfiguration getOffices =         
+        let sendEmailNotification (booking: Booking) =
+            let config = getEmailServiceConfiguration()        
+            let officeName =
+                let result = OfficeQueriesHandler.getAll getOffices
+                let officeReference = Guid.Parse(booking.Office.Id) |> OfficeId
+                match result with
+                | Ok offices ->
+                    offices
+                    |> List.find (fun (o:Office) -> officeReference.Equals(o.Id))
```

Comment:
List.find can fail, please handle the failure correctly

Reply:
To Do: tryFind applied to this List.find.